cli/cli_cn.md

```diff
@@ -0,0 +1,258 @@
+# Curveadm CLI 开发

```

Comment:
We should put these two document files in the root directory, at the same level as README.md, and we also need to add links to these two files in README.md. You can put the links in the `Contributing` section , used to prompt contributors to view.